### ISSUE TITLE
[4.4 dev 7] crash when duplicating light nodes

### ISSUE THREAD

**Issue by SlashScreen** [godotengine/godot#100972]
### Tested versions

Reproducible in  Godot Engine v4.4.dev7.mono.official (46c8f8c5c5874c7c56ea5b1384259de9402d3449).

### System information

Windows 10

### Issue description

The editor hangs and crashes when duplicating light nodes in the editor.
It reports the following:
```
CrashHandlerException: Program crashed with signal 11
Engine version: Godot Engine v4.4.dev7.mono.official (46c8f8c5c5874c7c56ea5b1384259de9402d3449)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
```
It proceeds to quit without dumping the backtrace. (Do I need to compile a version of the editor with debug enabled?)

### Steps to reproduce

1. Open a 3D scene. 
2. Add a light of some kind.
3. Attempt to duplicate it using the hierarchy editor.

As far as I can test, it only happens with lights.

### Minimal reproduction project (MRP)

[light_crash_mrp.zip](https://github.com/user-attachments/files/18279894/light_crash_mrp.zip)

**Comment by matheusmdx** [comment]
This issue was already reported and solved (see the pinned issue #100629). The fix will come in the next release.

**Comment by SlashScreen** [comment]
Ah, that didn't show up when I searched in the issues. Thanks.
